### ISSUE TITLE
Option to keep failed rips

### ISSUE THREAD

**Issue by BBaoVanC** [whipper-team/whipper#612]
I have a few very badly damaged discs that won't rip correctly. Once whipper exhaust its retries and the CRC's still don't match, it just gives up and deletes the track. It would be nice to have an option to keep the track even if it's not perfect so I at least have some digital version at all.

Additionally, it would be nice to have a feature to skip the test CRC (so it only does 1 rip, like the "Copy" option in EAC instead of "Test & Copy" which is what whipper does by default).

**Comment by MerlijnWajer** [comment]
Agreed, both options would be nice to have.

**Comment by pmoris** [comment]
Jumping in here to add that this would also be very useful for CDs that can't be ripped due to "copy controlled technology". Apparently one album in my collection has this: every rip attempt fails entirely because the test and copy CRCs seem to be unique every time, but I can't find a way to store any of the attempts using whipper.

**Comment by MerlijnWajer** [comment]
Agreed, I think this is helpful. PR https://github.com/whipper-team/whipper/pull/649 is almost ready for merging, we just need to figure out the exit code behaviour.
